### PR TITLE
Add Benjamin Cabé as doc maintainer

### PR DESCRIPTION
**Pull request by kartben**

### CODE REVIEW

File: MAINTAINERS.yml
Comment (stephanosio) [inline MAINTAINERS.yml:484]: You should add yourself as the first maintainer in the list so that the CI assigns any related PRs to you.

**Review by stephanosio — CHANGES_REQUESTED** [review]
* The commit title needs to be prefixed with a relevant area name (e.g. `MAINTAINERS: ...`).
* The commit message needs to contain a description of the changes.
* It also needs to contain a sign-off entry.

https://docs.zephyrproject.org/latest/contribute/guidelines.html#commit-guidelines

**Review by stephanosio — DISMISSED** [review]

File: MAINTAINERS.yml
Comment (gmarull) [inline MAINTAINERS.yml:501]: I think infra should be collaboration level first, until relevant changes are made in the area.

Comment (kartben) [reply MAINTAINERS.yml:501]: certainly fine with me

Comment (carlescufi) [reply MAINTAINERS.yml:501]: Can you update the PR @kartben?

**Review by gmarull — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Pull request merged**
